features/article_are_grouped_in_categories.feature

```diff
@@ -10,9 +10,9 @@ Background:
     | Technology    |
 
   And we have the following articles
-    | headline                            | category   |
-    | The sport team won something        | Sports     |
-    | Some programers made some new stuff | Technology |
+    | headline                            | category   | published |
+    | The sport team won something        | Sports     | true     |
+    | Some programers made some new stuff | Technology | true     |
```

Comment:
You can let the Factory deal with the `published` attribute if you are not using it in the scenario. Would lead to less code == less changed files == less work for you == less work for me ;-)

Reply:
Done, looks much cleaner now.